src/ipmi.c

```diff
@@ -138,43 +193,50 @@ static void sensor_read_handler(ipmi_sensor_t *sensor, int err,
     } else if (IPMI_IS_IPMI_ERR(err) &&
                IPMI_GET_IPMI_ERR(err) ==
                    IPMI_NOT_SUPPORTED_IN_PRESENT_STATE_CC) {
-      INFO("ipmi plugin: sensor_read_handler: Sensor %s not ready",
-           list_item->sensor_name);
+      INFO("ipmi plugin: sensor_read_handler: Sensor `%s` of `%s` not ready.",
+           list_item->sensor_name, st->name);
+    } else if (IPMI_IS_IPMI_ERR(err) &&
+               IPMI_GET_IPMI_ERR(err) == IPMI_TIMEOUT_CC) {
+      INFO("ipmi plugin: sensor_read_handler: Sensor `%s` of `%s` timed out.",
+           list_item->sensor_name, st->name);
     } else {
+      char errbuf[128] = {0};
```

Comment:
are you sure 128 is big enough? most other errbufs use 1024... I would also suggest replacing the magic # with a #define 

#define ERR_BUF_SIZE 1024

Reply:
That was inspired by OpenIPMI examlpes code:

https://github.com/wrouesnel/openipmi/blob/master/sample/waiter_sample.c#L285
https://github.com/wrouesnel/openipmi/blob/master/cmdlang/ipmish.c#L595
https://github.com/wrouesnel/openipmi/blob/master/sample/eventd.c#L53
https://github.com/wrouesnel/openipmi/blob/master/include/OpenIPMI/ipmi_err.h#L161

I agree with your suggestion, that would be better to remove the magic.

Reply:
The sources for `ipmi_get_error_string` return values are:

1) strerror() + 4 bytes of prefix ("OS: ")
2) ipmi_get_cc_string() (ipmi_ccodes[] array) + 6 bytes prefix - fits 128 bytes
3) rmcpp_error_codes[] + 7 bytes prefix  - fits 128 bytes
4) sol_error_codes[] + 5 bytes prefix    - fits 128 bytes
5) "Success (No error)" string           - fits 128 bytes
6) "Unknown: Unknown" string             - fits 128 bytes

The longest is strerror(). Will 1020 (1024 - 4) be enough for `strerror()`?